docs/docs/gatsby-on-linux.md

```diff
@@ -4,12 +4,10 @@ title: Gatsby on Linux
 
 # Linux
 
-> This is a TODO. Help our community expand it.
-
-> Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your pull request gets accepted.
-
```

Comment:
Pls, undo these changes, as WSL !== Linux. The idea is the same but the underlying things may differ a bit.

Reply:
I know it's different, it was just mentioned in #15529 for the "TODO" part to be removed, I will add it again. I have only made some small changes in this file.